src/items/structs.md

```diff
@@ -1,5 +1,41 @@
 # Structs
 
+> **<sup>Syntax</sup>**  
+> _Struct_ :  
+> &nbsp;&nbsp; &nbsp;&nbsp; _StructStruct_  
+> &nbsp;&nbsp; | _TupleStruct_  
+>  
+> _StructStruct_ :  
+> &nbsp;&nbsp; `struct`
+>   [IDENTIFIER]&nbsp;
+>   [_Generics_]<sup>?</sup>
+>   [_WhereClause_]<sup>?</sup>
+>   ( `{` _StructFields_<sup>?</sup> `}` | `;` )  
```

Comment:
Would it help to have this match the text and split this into _StructStruct_ and _UnitStruct_?

Reply:
I think I considered that when I first reviewed @brauliobz 's grammar and found a reason against it. Not sure anymore though. I'd rather not mess with the PR anymore than needed right now.